tests/integration/utils.sh

```diff
@@ -168,10 +167,24 @@ function container_run()
   fi
 }
 
+# Execute a command within a container.
+#
+# @container_options   Additional options for the 'podman container exec' command.
```

Comment:
Maybe rename this to something that makes it clearer that the options are specific to `podman container exec`.